resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -211,6 +228,10 @@ caws.download.thin_client=Downloading IDE thin client
 caws.environment.status=Dev Environment is currently: {0}
 caws.environment.view_pricing=Learn more about Dev Environment pricing
 caws.free.tier.subscription.storage=Other storage options are not currently available for your space. These require upgrading the billing tier.
+caws.getstarted.panel.description=Spend more time coding and less <br>time managing development <br>environments.
```

Comment:
it would be nice if these reflowed automatically instead of hardcoding the breaks